container/worker/run_openqa_worker.sh

```diff
@@ -1,18 +1,25 @@
 #!/bin/bash
-# shellcheck disable=SC2012
+# shellcheck disable=SC2012,SC2154
 set -e
+set -x
 
 if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
   OPENQA_WORKER_INSTANCE=1
 fi
 
-/root/qemu/kvm-mknod.sh
+if [[ -z $qemu_no_kvm ]] || [[ $qemu_no_kvm -eq 0 ]]; then
+  if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
+    /root/qemu/kvm-mknod.sh
 
-if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
-  usermod -a -G kvm _openqa-worker
-else
-  echo "Warning: /dev/kvm doesn't exist. If you want to use KVM, run the container with --device=/dev/kvm"
+    group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+    groupmod -g "$group" --non-unique kvm
+    usermod -a -G kvm _openqa-worker
+  else
+    echo "Warning: /dev/kvm doesn't exist. If you want to use KVM, run the container with --device=/dev/kvm"
+  fi
 fi
 
+qemu-system-x86_64 -S &
+kill $!
```

Comment:
can you explain this line please?

Reply:
It tests whether qemu starts and terminates with zero return code.

Reply:
yeah ok, that makes sense for testing but it is helpful for the user as well?

Reply:
I understand, if I am not wrong that with that, we are checking that this worker can execute jobs